packages/playground/wordpress-builds/build/build.js

```diff
@@ -79,8 +81,8 @@ if (args.wpVersion === 'nightly') {
 }
 
 if(!versionInfo.url) {
-	process.stdout.write(`WP version ${args.wpVersion} is not supported\n`);
-	process.stdout.write(await parser.getHelp());
+	logger.log(`WP version ${args.wpVersion} is not supported\n`);
```

Comment:
Why use the logger for this?

Reply:
It would, is that a problem?

My thinking was, let's use a single function to log all messages so that there is only one place `quiet` needs to be checked.

Reply:
Well, you typically expect the help message to explain what's wrong with the current command. Writing it to a file instead of stdout would be very confusing and we'd likely get a few issues like "Playground CLI doesn't work, there's no output".

Reply:
I see it now. Thanks for explaining!

I updated the PR now so that only server status updates like _Starting server_ use the logger.

Reply:
Thank you!